components/item/MetadataField.vue

```diff
@@ -33,13 +33,29 @@
             />
           </li>
         </template>
+        <template v-else-if="typeof fieldData.url !== 'undefined'">
+          <li
+            :key="index"
+            :lang="langMappedValues.code"
+            data-qa="literal value"
```

Comment:
Is this correct? As in this case, it is not the literal value, right?

Reply:
as discussed we'll change to 'entity link'